tensorboard/plugins/interactive_inference/utils/platform_utils.py

```diff
@@ -188,7 +187,7 @@ def call_servo(examples, serving_bundle):
 
   if serving_bundle.use_predict:
     request.inputs[serving_bundle.predict_input_tensor].CopyFrom(
-        tensor_util.make_tensor_proto(
+        tf.compat.v1.make_tensor_proto(
```

Comment:
@nfelt any thoughts on this? Without this change, witwidget pip package needs to copy over all the transitive dependencies which seem intractable. Either the package can have hard dependency on TB or TF. It really is not ideal in either form.

Reply:
many part of the plugin have deps on TF itself so a pip package dep on TF makes sense. deps include: making sprite images using TF ops, using tf.data.Dataset, tf record reader.